sdks/typescript/src/v1/client/worker.ts

```diff
@@ -0,0 +1,123 @@
+import { WorkerLabels } from '@hatchet/clients/dispatcher/dispatcher-client';
+import { InternalHatchetClient } from '@hatchet/clients/hatchet-client';
+import { V0Worker } from '@clients/worker';
+import { Workflow as V0Workflow } from '@hatchet/workflow';
+import { Workflow } from '../workflow';
+
+/**
+ * Options for creating a new hatchet worker
+ * @interface CreateWorkerOpts
+ */
+export interface CreateWorkerOpts {
+  /** Maximum number of concurrent runs on this worker */
+  slots?: number;
+  /** Array of workflows to register */
+  workflows?: Workflow<any, any>[] | V0Workflow[];
+  /** Worker labels for affinity-based assignment */
+  labels?: WorkerLabels;
+  /** Whether to handle kill signals */
+  handleKill?: boolean;
+  /** @deprecated Use slots instead */
+  maxRuns?: number;
```

Comment:
I've been hard-removing these params instead of deprecating - WDYT of that? I figure that since this is a major version, we're within our bounds to do that, and it might be a while before we have another opportunity

Reply:
typescript respects the @deprecated tag in most IDEs much better than python in my experience... personally I'm a bigger fan of backwards compatibility and eventually removing features.

by having the v0 client in ts my thoughts were we can eventually rip and replace the v0 client with a long term v1 client and start printing warnings before we fully remove the feature